scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -594,18 +622,15 @@ class FormatWriter(formatOps: FormatOps) {
           if (matches == 0 || doubleNewline || !locationIter.hasNext) {
             var column = 0
             val columns = minMatches
+
             while (column < columns) {
-              val blockWithWidth = block.map { line =>
```

Comment:
do we need any of the changes here and below, after the `shift` adjustment above?